man/systemd-keyutil.xml

```diff
@@ -109,6 +110,15 @@
         <xi:include href="version-info.xml" xpointer="v258"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--content=<replaceable>PATH</replaceable></option></term>
+
+        <listitem><para>Input data that corresponds to the PKCS#1 signature for the <command>pkcs7</command>
+        command.</para>
```

Comment:
maybe say "for generating inline signatures" or so

Reply:
updated